src/onboarding/steps.ts

```diff
@@ -0,0 +1,214 @@
+import { BIOMETRY_TYPE } from 'react-native-keychain'
+import { initializeAccount } from 'src/account/actions'
+import {
+  choseToRestoreAccountSelector,
+  recoveringFromStoreWipeSelector,
+} from 'src/account/selectors'
+import {
+  numberVerifiedCentrallySelector,
+  skipVerificationSelector,
+  supportedBiometryTypeSelector,
+} from 'src/app/selectors'
+import { setHasSeenVerificationNux } from 'src/identity/actions'
+import * as NavigationService from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
+import { store } from 'src/redux/store'
+
+export const FIRST_ONBOARDING_SCREEN = Screens.NameAndPicture
+export const END_OF_ONBOARDING_SCREEN = Screens.WalletHome
+
+interface NavigatorFunctions {
+  navigate: typeof NavigationService.navigate | ((screen: Screens) => void)
+  popToScreen: typeof NavigationService.popToScreen | ((screen: Screens) => void)
+  navigateHome: typeof NavigationService.navigateHome | (() => void)
+  navigateClearingStack:
+    | typeof NavigationService.navigateClearingStack
+    | ((screen: Screens) => void)
+}
+
+interface GetStepInfoProps {
+  firstScreenInStep: Screens
+  navigator: NavigatorFunctions
+  dispatch: (action: any) => void
+  props: OnboardingProps
+}
+
+export interface OnboardingProps {
+  recoveringFromStoreWipe: boolean
+  choseToRestoreAccount: boolean | undefined
+  supportedBiometryType: BIOMETRY_TYPE | null
+  skipVerification: boolean
+  numberAlreadyVerifiedCentrally: boolean
+}
+
+export function onboardingPropsSelector(state: RootState): OnboardingProps {
```

Comment:
Maybe add a docstring explaining that any data that parameterizes onboarding screen logic must be included in this function's result?